app/src/main/java/com/dap/meau/Model/PetModel.java

```diff
@@ -31,6 +26,16 @@ public PetModel(String name, String gender, String age, String postage, String c
         this.vaccinated = vaccinated;
     }
 
+    public PetModel(String name, String gender, String age, String postage, String city, String imageUrl, String uid) {
+        this.name = name;
+        this.gender = gender;
+        this.age = age;
+        this.postage = postage;
+        this.city = city;
+        this.imageUrl = imageUrl;
+        this.uid = uid;
```

Comment:
Colocar a variável uid na constructor anterior, para ele não ter que pegar a uid de todos os donos sem necessidade

Reply:
Ao chamar as _cardviews_ no caso, certo?

Reply:
Colocar a variável uid somente quando for montar o perfil completo do animal, no caso quando se clicar em um cardview específico.